hugegraph-core/pom.xml

```diff
@@ -174,6 +174,23 @@
             <artifactId>fastutil</artifactId>
             <version>8.1.0</version>
         </dependency>
+        <dependency>
```

Comment:
add empty line and some comment like `<!-- jwt auth token TODO: move to auth pom -->`